main.go

```diff
@@ -351,8 +351,15 @@ func runGcloud(runner *Environ, workspace string, vargs GAE) error {
 		}
 	}
 
-	// add the app.yaml location
-	args = append(args, "./app.yaml")
+	switch {
+	// hook in the apropro yaml file
+	case vargs.DispatchFile != "":
+		args = append(args, "./dispatch.yaml")
```

Comment:
Question: you use `"./dispatch.yaml"` `cron.yaml` etc here which relies on the step below that moves the files into the right place.
However, with `gcloud app deploy` you can pass absolute file names so why not pass `vargs.Dir+vargs.DispatchFile` as the arg and start deprecating the `setupFile()` stuff?

Reply:
At this point, I'm looking to make the smallest change for max impact while keeping backwards compatibility.

I think it'll definitely be worth simplifying all this bit in the future when the `appcfg.py` is gone for good.

I think in the future, we'd probably be able to string up multiple yaml files of different styles and the tool _should_ stack em up and run them together.

Reply:
👍 

and another 👍 for handling multiple yaml files in one drone block (in the future).

Reply:
If we ever want to start deprecating `appcfg`, I would pull that out into a separate plugin `drone-appcfg` and strip down `drone-gae` to use just `gcloud`